rocketpy/rocket/aero_surface.py

```diff
@@ -244,6 +260,14 @@ def length(self, value):
         self.evaluate_center_of_pressure()
         self.evaluate_nose_shape()
 
+    @property
+    def power(self):
+        return self._power
+
+    @power.setter
+    def power(self, value):
+        self._power = value
+
```

Comment:
This setter should behave similar to the `bluffness` setter

Reply:
Thanks for the review! I will make the suggested changes tonight or tomorrow. With respect to the `bluffness` parameter, should I raise an error if it is not `None` when passing a power series nose cone?

Reply:
I think ignoring the bluffness and raising a warning would be best